### ISSUE TITLE
Failure to build on FreeBSD: mismatched types in limits.go

### ISSUE THREAD

**Issue by rhclayto** [GoogleCloudPlatform/cloudsql-proxy#252]
When trying to `go get` this package on FreeBSD 11.2, I get the following:

```sh
user@appserver-0:~/go # go get github.com/GoogleCloudPlatform/cloudsql-proxy/cmd/cloud_sql_proxy
# github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/limits
src/github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/limits/limits.go:49:14: invalid operation: rlim.Cur >= wantFDs (mismatched types int64 and uint64)
src/github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/limits/limits.go:62:14: invalid operation: rlim.Max < wantFDs (mismatched types int64 and uint64)
src/github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/limits/limits.go:66:13: cannot use wantFDs (type uint64) as type int64 in assignment
src/github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/limits/limits.go:67:13: cannot use wantFDs (type uint64) as type int64 in assignment
src/github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/limits/limits.go:74:11: cannot use wantFDs (type uint64) as type int64 in assignment
```

**Comment by rhclayto** [comment]
I discovered what was causing the problem. FreeBSD uses int64 for its rlimit values, whereas this code is expecting uint64. I submitted a pull request fixing it at https://github.com/GoogleCloudPlatform/cloudsql-proxy/pull/253 .

**Comment by broady** [comment]
Fixed in #253.
